src/test/groovy/org/gradle/profiler/mutations/support/ProjectCombinationsTest.groovy

```diff
@@ -1,12 +1,15 @@
 package org.gradle.profiler.mutations.support
```

Comment:
Shall we move this to a level higher? Or merge the test into `ApplyProjectDependencyChangeMutatorTest`?

Reply:
Done